front/src/config/i18n/en.json

```diff
@@ -243,6 +244,10 @@
         "addButton": "+",
         "substractButton": "-"
       },
+      "devices": {
+        "editDeviceFeaturesLabel": "Select the devices you want to display. Only lights / sensors are showed here.",
```

Comment:
are we only supporting sensors & on/off switch?

Reply:
IDK, If have copied the same behaviour as DevicesInRoom

Reply:
Indeed, this is an old comment, it can be removed